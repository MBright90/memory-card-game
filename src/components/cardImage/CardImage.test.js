import { render } from '@testing-library/react'
import React from 'react'

import CardImage from './CardImage'

describe('CardImage', () => {
  test('Renders using the correct url', () => {
    const imageUrl = 'www.example.com/jarJarBinks'

    render(<CardImage imageUrl={imageUrl} />)
    const cardImage = screen.getByRole('img')
    expect(cardImage).toHaveStyle(`background-image: url(${imageUrl})`)
  })
})
