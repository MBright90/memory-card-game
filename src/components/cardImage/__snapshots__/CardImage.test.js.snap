// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CardImage renders to match snapshot 1`] = `
<div>
  <div
    class="cardImage"
    role="img"
    style="background-image: url(www.example.com/jarJarBinks);"
  />
</div>
`;
